src/index.js

```diff
@@ -12,3 +12,22 @@ export function enableBatching(reduce) {
 		return reduce(state, action);
 	}
 }
+
+export function batchDispatchMiddleware(store) {
+	function dispatchChildActions(store, action) {
+		if(action.meta && action.meta.batch) {
+			action.payload.map(function(childAction) {
```

Comment:
Small thing - this should really be `forEach`, rather than `map`, since we aren't `return`ing or using the value. That way, it conveys the "side-effecty-ness" of it a bit better.